### ISSUE TITLE
Middle Mouse Button - open tab without focus

### ISSUE THREAD

**Issue by philiprhoades** [cxw42/TabFern#160]
I am slowly getting used to single-clicking for opening tabs rather than selecting them - I would still prefer to be able to select a range with with single clicking plus SHIFT and / or CTRL keys for bulk actions - but in advance of that, it would be good to be able to open tabs with the Middle Mouse Button with changing focus so that I can selectively open a bunch of tabs quickly without having to keep changing back to the TF window after each tab.  ATM, using the Middle Mouse Button opens another tab of the TF window itself.

**Comment by cxw42** [comment]
I agree, and am officially marking "MMB => new TF window" as a bug.  See #29.

**Comment by cxw42** [comment]
@philiprhoades Check out the latest on the issue35 branch - first version of MMB click seems to be working :D

**Comment by cxw42** [comment]
- Looks like I still need to work on the event a bit - clicking on the space next to the name (as opposed to the name itself) triggers a scroll.  Maybe the event handler needs to be applied farther down?  (Hopefully not to each `<li>`!

- If you have two windows open, plus the TF window, the current middle-click will not raise the back Chrome window above the front Chrome window.  Should it?

**Comment by cxw42** [comment]
This is all working as of current `main` :)
